tests/unit/test_prometheus_metrics.py

```
@@ -241,14 +241,26 @@ def test_aroi_state_and_counts(tmp_path, make_relay, make_relay_set, sample_dns_
+    # proof_type_family. All test fixture relays declare ciissversion:2
+    # with rsa proof types via the contact string.
+    assert 'aeo1_aroi_relay_state{fingerprint="AAAA",familyid="",state="configured_checked_valid",ciissversion="2",proof_type_family="rsa"} 1' in content
+    assert 'aeo1_aroi_relay_state{fingerprint="BBBB",familyid="",state="configured_checked_invalid",ciissversion="2",proof_type_family="rsa"} 1' in content
+    assert 'aeo1_aroi_relay_state{fingerprint="CCCC",familyid="",state="configured_unchecked",ciissversion="2",proof_type_family="rsa"} 1' in content
+    # DDDD has plain contact (no ciissversion) — labels: none/none.
+    assert 'aeo1_aroi_relay_state{fingerprint="DDDD",familyid="",state="not_configured",ciissversion="none",proof_type_family="none"} 1' in content
     assert 'aeo1_aroi_relays_count{state="not_configured"} 1' in content
     assert 'aeo1_aroi_relays_count{state="configured_unchecked"} 1' in content
     assert 'aeo1_aroi_relays_count{state="configured_checked_invalid"} 1' in content
     assert 'aeo1_aroi_relays_count{state="configured_checked_valid"} 1' in content
+    # B7.2: aeo1_aroi_relays_count_by_version emits all 16 (4 state × 4
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)